setup.py

```diff
@@ -4,18 +4,21 @@
 with open('README.md') as f:
     readme = f.read()
 
-with open('LICENSE') as f:
-    license_ = f.read()
-
 setup(
     name='cellmlmanip',
     version='0.1.0',
     description='CellML loading and model equation manipulation',
     long_description=readme,
+    long_description_content_type="text/markdown",
```

Comment:
We're using `'` in the rest of the arguments

Reply:
sure